gateway/handlers/createhandler.go

```diff
@@ -104,14 +121,19 @@ func makeSpec(request *requests.CreateFunctionRequest, maxRestarts uint64, resta
 				Delay:       &restartDelay,
 			},
 			ContainerSpec: swarm.ContainerSpec{
-				Image:  request.Image,
-				Labels: labels,
+				Image:   request.Image,
+				Labels:  labels,
+				Secrets: secrets,
 			},
 			Networks: nets,
 			Placement: &swarm.Placement{
 				Constraints: constraints,
 			},
 		},
+		Annotations: swarm.Annotations{
```

Comment:
Repeat of code already present lines 113-116

Reply:
Sorry about that, those repeat lines must have snuck in we I rebased it last week.